detectors/detectron2.py

```diff
@@ -0,0 +1,90 @@
+'''
+Performs vehicle detection using models created with the YOLO (You Only Look Once) neural net.
+https://pjreddie.com/darknet/yolo/
+'''
+
+import ast
+import os
+import cv2
+import torch
+
+import detectron2
+from detectron2.utils.logger import setup_logger
+setup_logger()
+
+# import some common libraries
+import numpy as np
+import random
+
+# import some common detectron2 utilities
+from detectron2.engine import DefaultPredictor
+from detectron2.config import get_cfg
+from detectron2.utils.visualizer import Visualizer
+from detectron2.data import MetadataCatalog
+
+with open(os.getenv('DETECTRON2_CLASSES_PATH'), 'r') as classes_file:
```

Comment:
Please add these environment variables to .env.example

Reply:
This are the required .env variables:

# DETECTRON2
DETECTRON2_CONFIDENCE_THRESHOLD=0.8
DETECTRON2_CONFIG_PATH="./data/detectors/detectron2/model_final_af9cf5.cfg"
DETECTRON2_WEIGHTS_PATH="./data/detectors/detectron2/model_final_af9cf5.pkl"
DETECTRON2_NUM_CLASSES=8
DETECTRON2_CLASSES_PATH="./data/detectors/detectron2/classes.txt"
DETECTRON2_CLASSES_OF_INTEREST_PATH="./data/detectors/detectron2/classes_of_interest.txt"